test/extended/templates/templateservicebroker_e2e.go

```diff
@@ -72,6 +75,12 @@ var _ = g.Describe("[templates] templateservicebroker end-to-end test", func() {
 		})
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it should not be required
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "view", cli.Username(), "-n", "openshift").Execute()
```

Comment:
the current prototype is not enforcing the setting of groups ... the request as to manually provide them ... once the SC is actually authenticating with openshift master, and obtaining the fully populated user.Info, then I would think that any groups the user is associated with would be set